<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1527</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>737</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="goToBeginning">
         <property name="text">
          <string>|&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="skipBackward">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="precedentFrame">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read">
         <property name="text">
          <string>lire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause">
         <property name="text">
          <string>pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextFrame">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="skipForward">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goToEnd">
         <property name="text">
          <string>&gt;|</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>441</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="load_video">
       <property name="text">
        <string>Charger une vidéo</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="video">
       <item>
        <widget class="QLabel" name="label_video">
         <property name="text">
          <string>Video : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentvideo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>D:/programme/video.mp4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="objets">
       <item>
        <widget class="QLabel" name="label_obs">
         <property name="text">
          <string>Objets :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="liste_objets">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_object">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ajouter...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rename_object">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>renommer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_object">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sequences">
       <item>
        <widget class="QLabel" name="label_sequences">
         <property name="text">
          <string>Séquences de l'objet :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="liste_sequences"/>
       </item>
       <item>
        <widget class="QPushButton" name="new_sequence">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ajouter...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rename_sequence">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>renommer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_sequence">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="framedebut">
       <item>
        <widget class="QLabel" name="label__frame_debut">
         <property name="text">
          <string>Frame de début de la séquence : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="frame_debut">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="frameinit">
       <item>
        <widget class="QLabel" name="label_frame_init">
         <property name="text">
          <string>Frame d'initialisation du masque : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initMask">
         <property name="text">
          <string>Initialiser masque</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="frame_init">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="framefin">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_frame_fin">
         <property name="text">
          <string>Frame de fin de la séquence : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="frame_fin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>749</width>
       <height>519</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>999999</width>
          <height>999999</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../screen apex.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="idFrame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>165</width>
      <height>186</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="mask_sequence">
       <property name="text">
        <string>Traiter la séquence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mask_all">
       <property name="text">
        <string>Traiter la vidéo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_object_mask">
       <property name="text">
        <string>Exporter les 
masques de l'objet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_fused_masks">
       <property name="text">
        <string>Exporter les 
 masques fusionnés</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="display_bbox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>158</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cacher les bbox</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tracker">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="debug">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>710</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>debug</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>50</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="list_mask"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="edit_mask">
         <property name="text">
          <string>créer/modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_mask">
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Aller à la frame n° : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="frameToGo"/>
     </item>
     <item>
      <widget class="QPushButton" name="goToButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1527</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadvideo">
   <property name="text">
    <string>Load video</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>goToBeginning</tabstop>
  <tabstop>goToEnd</tabstop>
  <tabstop>nextFrame</tabstop>
  <tabstop>precedentFrame</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
